blocks/large-promo-block/features/large-promo/default.jsx

```diff
@@ -99,12 +139,145 @@ const LargePromoItem = ({ customFields, arcSite }) => {
     }`,
 		}) || null;
 
-	return <LargePromoPresentation content={content} {...customFields} />;
+	// const { id } = useFusionContext();
```

Comment:
do we need this id? if not we can delete this comment

Reply:
comments have been removed.